dv/uvm/core_ibex/riscv_dv_extension/riscvOVPsim.ic

```diff
@@ -1,5 +1,5 @@
 # riscOVPsim configuration file converted from YAML
---variant RV32IMC
+--variant RV32IMCB
```

Comment:
```suggestion
--variant RV32I
--override riscvOVPsim/cpu/add_Extensions=MCB
```

Structuring the supported extensions in this way is required for OVPsim to recognize bitmanip instructions and successfully run, otherwise it will still fail.

Reply:
:+1: thanks for the suggestion